import_package.md

```diff
@@ -0,0 +1,280 @@
+# Import package
+
+* An import package is a .zip file containing a **manifest.yaml** that describes additional 
+  contents and actions that should be carried out when processing the package.
+* A valid import package must contain all resource files that are referenced in the 
+  **manifest.yaml**.
+
+## Meta-data
+
+* `apiVersion` is set to `v1beta1` 
+
+## Tasks
```

Comment:
Are these Keptn tasks as specified in the shipyard?  If not, do we really want to use this same term for something else?  What about calling them "actions"?

Reply:
These Task are specific to the import Endpoint and have nothing to do with the *shipyard.yaml*. You can have two different types of tasks, an API action that executes something and a resource task that upload resources to Keptn.

Reply:
That's what I thought.  Which is why I am thinking that it would be nice if we could find a term other than "tasks".  Using the same term for different things makes for a lot of confusion -- like we have with the term "service".

Reply:
@StackScribe renaming "tasks" to something else here is unfortunately not an option. I understand that it can cause confusion, but at the same time, "task" is not a special word for Keptn. It's a very universal English word, such as "start" and "stop". Also, I believe that by having a good documentation/spec, confusion can be avoided.